randovania/games/fusion/logic_database/Sector 4 AQA.txt

```diff
@@ -1394,16 +1409,25 @@ Extra - room_id: [34]
   > Other to Security Access
       Any of the following:
           Can Use Any Bombs
-          Gravity Suit and Screw Attack and Knowledge (Beginner)
+          All of the following:
+              Gravity Suit
+              Any of the following:
+                  Screw Attack and Knowledge (Beginner)
+                  # Small Jump to break block
+                  Speed Booster and Movement (Intermediate)
 
 > Other to Security Access; Heals? False
   * Layers: default
   * Open Passage to Security Access/Other to Security Bypass
   * Extra - door_idx: (82,)
   > Other to Level 4 Security Room
-      Any of the following:
-          Can Use Any Bombs
-          Gravity Suit and Morph Ball and Screw Attack and Knowledge (Beginner)
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Gravity Suit and Screw Attack and Knowledge (Beginner)
+              All of the following:
+                  Can Use Any Bombs
+                  Mid-Air Morph (Beginner) or Can Use Springball
```

Comment:
![grafik](https://github.com/user-attachments/assets/ee72b06c-824a-423b-83a1-a4c06b865574)
mid air morph is needed to go through the tunnel here because you cant ledge grab it, right?

Reply:
correct, hole is only one tile up from the ground

Reply:
bombs + grav also works